controller/src/main/java/io/pravega/controller/store/stream/Stream.java

```diff
@@ -522,6 +522,15 @@
      */
     CompletableFuture<Map<UUID, ActiveTxnRecord>> getActiveTxns(OperationContext context);
 
+    /**
+     * Api to retrieve List transaction in completed(COMMITTED/ABORTED) state
```

Comment:
Missing space after "completed". 
Api -> API (is an acronym).

Reply:
Done as suggested.